daq/session_server.py

```diff
@@ -101,18 +112,42 @@ def _init_session(self, request):
     def _session_stream(self, request):
         device_mac = request.device_mac
         while True:
+            self._last_streamed[device_mac] = time.time()
```

Comment:
I think this line should go after the next line like this
<pre>
item = self._return_queues[device_mac].get()
self._last_streamed[device_mac] = time.time()
</pre>
since queue.get is a blocking call

Reply:
Fixed.